c7n/filters/core.py

```diff
@@ -90,6 +98,7 @@ def intersect(x, y):
     'less-than': operator.lt,
     'glob': glob_match,
     'regex': regex_match,
+    'regex-cs': regex_case_sensitive_match,
```

Comment:
thoughts on renaming to `regex-case`?... cs feels a bit ambigious without context.